### DISCUSSION TITLE
Self capacitance

### DISCUSSION THREAD

**Discussion by jhgorse** [thliebig/openEMS-Project#144] [General]
Greetings,

Can this predict the SPICE lump discrete circuit values, such as self capacitance, of a modeled air gap coil?

Cheers,
Joe

**Comment by biergaizi** [comment]
In principle, yes. If you create a 3D model of the inductor coil, it's certainly possible to measure its S-parameters, thus allowing the extraction of its interwinding capacitance. The main limitation of FDTD is that it works in the high frequency regime, such as 100 MHz or 1 GHz, it's unsuitable for low-frequency uses - For capacitance extraction of a capacitor, the frequency used makes little difference in general because it's an electrostatic phenomenon, doing it at RF is fine in general.

But the situation is more complex in inductor coil analysis. At RF, the inductor coil is not really coil but a transmission line, there exists multiple resonance frequencies, so it cannot be represented using solely lumped-element theory [0]. You cannot really define a unique "self-capacitance" at RF. According to [1]:

> [...] compared with theories attributing self-capacitance to the capacitance between adjacent turns, and also with transmission-line theories. The inter-turn capacitance approach is found to have *no predictive power*. 

These RF behaviors will show up in openEMS simulations too. If your goal is to design microwave circuits, this is exactly what you need. But if your goal is to simply know its lumped capacitance at DC or a very-low frequency, FDTD's results can be very confusing - but it's not a showstopper, if you already know how to do real-world measurements of inductors with a VNA at RF, the same technique should work in FDTD simulations as well. 

[0] https://g3ynh.info/zdocs/magnetics/appendix/self_res/gallery.html

[1] https://g3ynh.info/zdocs/magnetics/appendix/self_res/self-res.pdf

**Comment by jhgorse** [comment]
biergaizi,

Thank you for writing. The thought is to design and reproduce the Magnetically Couple Resonators for power transmission such as those found here:
https://theisclab.com/docs/2010_TIE_MCR.pdf

In the paper, they vary frequency to optimize coupling. Unless we can stay below FCC power limits, that is unlikely to be a viable approach for most use cases. The other options are to (1) physically rotate the drive coil with respect to the tx/rx coil and (2) tune network impedances with a matching network.

The frequencies used were from 8-12 Mhz. So it may not be high enough for openEMS.

At university we used ADS to simulate and design these kinds of systems before fabrication and tuning. It would be useful to set up some rough simulations to get decent numbers to narrow the search space for the coils.

Cheers,
Joe

**Reply by biergaizi** [reply]
Unfortunately, the fundamental limitation of conventional FDTD is that the timestep used must be small enough to resolve *both* the smallest physical structure *and* the highest frequency component. When simulating an object with small details at low frequencies, the require timesteps become prohibitively large (e.g. nanosecond to picosecond timesteps are used to simulate a 1 MHz signal). Thus, FDTD in its unmodified form is unsuitable for electrically-small structures. There are some modifications to FDTD that would allow such uses, but those are not included in openEMS so far. For example, see:

* Acceleration of FDTD calculation of EM fields due to loop antennas used for MHz band wireless transfer system placed near human body, Keita Asano, Toru Uno, and Takuji Arima, IEICE Communications Express, Vol.6, No.6, 325–330  https://www.jstage.jst.go.jp/article/comex/6/6/6_2016SPL0036/_pdf/-char/ja
